guides/v2.1/test/integration/integration_test_execution.md

```diff
@@ -314,8 +329,7 @@ The only difference in the run configuration is that the integration test `phpun
 
```

Comment:
I can't leave a comment on line 324, but it (`Use the integration test configuration file`) could probably be an H3 instead of an H4.

Reply:
Thanks. Headings were messy on this page.